packages/jest-circus/src/legacy_code_todo_rewrite/jest_adapter_init.js

```diff
@@ -102,7 +102,7 @@ export const runAndTransformResultsToJestFormat = async ({
   globalConfig: GlobalConfig,
   testPath: string,
 }): Promise<TestResult> => {
-  const runResult = await run();
+  const runResult = await run(config);
```

Comment:
i'm really not sure about passing the config to `jest-circus` world, that couples it pretty tightly to the rest of `jest`
would you be ok dispatching an action, like i did on line 73 in this file?

Reply:
Done.